htdocs/comm/propal/card.php

```diff
@@ -2298,7 +2298,7 @@
 					$langs->load("contracts");
 
 					if ($user->rights->contrat->creer) {
-						print '<div class="inline-block divButAction"><a class="butAction" href="' . DOL_URL_ROOT . '/contrat/card.php?action=create&amp;origin=' . $object->element . '&amp;originid=' . $object->id . '&amp;socid=' . $object->socid . '">' . $langs->trans('AddContract') . '</a></div>';
+						print '<div class="inline-block divButAction"><a class="butAction" href="' . DOL_URL_ROOT . '/contrat/card.php?action=create&amp;origin=' . $object->element . '&amp;originid=' . $object->id . '&amp;ref_customer=' . $object->ref_client . '&amp;socid=' . $object->socid . '">' . $langs->trans('AddContract') . '</a></div>';
```

Comment:
Here object->ref_customer is the reference on customer side of the order. If you provide it as parameter, this ref will be into the field of reference of contract that may be 2 different things.
The proposal should be linked to contract, so the ref can be visible on the contract. But i don't think we must share a reference of an object into a field dedicated for a reference of another. This may be useful for tracking, but we shoul already have this tracking link with link between contract and source proposal.